### PR TITLE
pool_status: add node info payload with pool status

### PR DESCRIPTION
**Pull request by jamesaimonetti**

### CODE REVIEW

File: core/kazoo_globals/src/kz_nodes.erl [hunk 1]
```diff
@@ -872,3 +900,25 @@ maybe_add_zone(#kz_node{zone=Zone, broker=B}, #state{zones=Zones}=State) ->
         'undefined' -> State#state{zones=[{Broker, Zone} | Zones]};
         _ -> State
     end.
+
+-spec node_info() -> kz_json:object().
+node_info() ->
+    kz_json:from_list(
+      props:filter_undefined(
```
Comment (fenollp) [inline core/kazoo_globals/src/kz_nodes.erl]: `filter_undefined` a `{ne_binary(), ne_binary()}`?

**Comment by lazedo** [comment]
@jamesaimonetti can we iterate thru kz_amqp_sup children , filtering for pools ? not required, only if it's easy to add it.

**Comment by jamesaimonetti** [comment]
@lazedo added

**Comment by jamesaimonetti** [comment]
@lazedo rebased

**Pull request merged**